moneo.py

```diff
@@ -113,7 +113,7 @@ def jobID_update(self):
         logging.info(out)
         logging.info('Job ID updated to ' + self.args.job_id + ". Hostfile: " + self.args.host_file)
 
-    def deploy_worker(self, hosts_file, max_threads=16):
+    def deploy_worker(self, hosts_file, max_threads=16):    # noqa: C901
```

Comment:
I'm guessing we hit a complexity limit here?

Reply:
wrt lint

Reply:
Yea, and I prefer to refactor the moneo.py and publisher-related config later.